content/llm/speaking-bots-all.md

```
@@ -60,6 +34,32 @@ This will:
 
 <APIPage document={"./speaking-bots-openapi.json"} operations={[{"path":"/bots/{bot_id}","method":"delete"}]} webhooks={[]} hasHead={false} />
 
+---
+
+## index
+
+### Source: ./content/docs/speaking-bots/reference/index.mdx
+
+# Speaking Bots API Reference
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)